### DISCUSSION TITLE
"Overview" window does not respect setting of "touchpad_sensitivity"

### DISCUSSION THREAD

**Discussion by qingshuizheng** [ahrm/sioyek#623] [Q&A] [answered]
Hi @ahrm , hope you are doing great!

It seems "touchpad_sensitivity" only applies to the regular window but not "Overview" window?

I've toned it down to `touchpad_sensitivity  .5` in prefs_user.config, and it works great in regular window, but scrolling in "Overview window" is still as fastest as crazy. 

Am i missing something here?

Thanks!
Zheng

**Comment by ahrm** [comment, marked as answer]
It should also be applied to overview window. Which version of sioyek are you using?

**Reply by qingshuizheng** [reply]
![Kapture 2023-02-06 at 04 57 51](https://user-images.githubusercontent.com/78839107/216943275-936e4b3b-ddc0-4d02-bdab-41eada3cc748.gif)

I record a gif for demostration:
1. `touchpad_sensitivity  .5` in prefs_user.config
2. i scroll the overview window and main window in the same way, but the scrolling in overview is behaving like `touchpad_sensitivity` is set to 1.0

Env: MacOS Big Sur (11.6)
Sioyek version:  v0.31.426, from here: https://github.com/hexomancer/sioyek/releases/download/v0.31.426/sioyek-release-mac.zip

Thanks for the quick reply.

**Reply by ahrm** [reply]
Added the ability to separately modify scrollview sensitivity in 94a1d743c8995326d54f96940c885b086d0f6b4e:

```
scrollview_sensitivity 0.1
```

**Reply by qingshuizheng** [reply]
Tested. Works great! Many many thanks. 😄
